md/use-gzip-compression.md

```diff
@@ -83,7 +40,7 @@ Connector configuration:
 After you modify the file, restart your application server and test with the following `curl` command:
 `curl -I -H 'Accept-Encoding: gzip' http://`_`ip_address:port`_`/bonita/login.jsp`
 
-Check that the header returned contains the line `Content-Encoding: gzip`. For example, on a WildFly system the output will be similar to this:
+Check that the header returned contains the line `Content-Encoding: gzip`. The output should be similar to this:
```

Comment:
We whould put a Tomcat example instead (if different from WildFly)

Reply:
done